src/Agent.Worker/Container/DockerCommandManager.cs

```diff
@@ -191,12 +191,7 @@ public async Task<List<string>> DockerPS(IExecutionContext context, string optio
 
         public async Task<int> DockerNetworkCreate(IExecutionContext context, string network)
         {
-            PlatformUtil.OS containerHostOS = PlatformUtil.HostOS;
-            if (context.StepTarget() != null)
-            {
-                containerHostOS = context.StepTarget().ImageOS;
-            }
-
+            var usingWindowsContainers = context.Containers.Where(x => x.ExecutionOS != PlatformUtil.OS.Windows).Count() == 0);
             var networkDrivers = await ExecuteDockerCommandAsync(context, "info", "-f \"{{range .Plugins.Network}}{{println .}}{{end}}\"");
             if (containerHostOS == PlatformUtil.OS.Windows && networkDrivers.Contains("nat"))
```

Comment:
did you mean for this if to use 'usingWindowsContainers'?

Reply:
VS Code didn't save my file -_-

Fixed